docs/data_ingestion.md

```diff
@@ -11,6 +12,22 @@ This guide provides more details for using the `prepdocs` script to index docume
   - [Removal of documents](#removal-of-documents)
   - [Scheduled indexing](#scheduled-indexing)
 
+## Supported document formats
+
+In order to ingest a document format, we need a tool that can turn it into text. By default, use Azure Document Intelligence (DI in the table below), but we also have local parsers for several formats. The local parsers are not as sophisticated as Azure Document Intelligence, but they can be used to decrease charges.
+
+| Format | Manual indexing                      | Integrated Vectorization |
+| ------ | ------------------------------------ | ------------------------ |
```

Comment:
@mattgotteiner Sorry, I ended up adding a section to a README about "Supported document formats" after sending this for review. What formats are supported by integrated vectorization? Is it truly this list here? https://learn.microsoft.com/en-us/azure/search/search-howto-indexing-azure-blob-storage#supported-document-formats

Reply:
yes this is the correct list.

Reply:
Great, adding link